pointers/pointers.c

```diff
@@ -53,7 +70,42 @@ void string_copy(char *x, char *y)
 */
 int string_compare(char *m, char *n)
 {
-
```

Comment:
Perhaps it will be more readable if you make declare and initialize different variables for each loop? Either way, great job on the progress given the amount of time you had.